src/App.js

```diff
@@ -56,28 +57,33 @@ class App extends React.Component {
       <div className="App">
         <RecoilRoot>
           <Router>
-            <SideNav collapsed={this.state.navCollapsed} toggleNav={this.toggleNav} />
-            <main
-              onClick={(e) => {
-                if (!this.state.navCollapsed) {
-                  this.closeNav()
-                  e.preventDefault()
-                  return
-                }
-              }}
-            >
-              <Header toggleNav={this.openNav} />
-              <Switch>
-                {Routes.map((route, key) => {
-                  let children = route.component
-                  if (route.requiresLogin && !this.props.user) {
-                    children = <Redirect to="/login" />
-                  }
-                  return <Route key={key} path={route.path} exact={route.exact} children={children}></Route>
-                })}
-              </Switch>
-              <Footer />
-            </main>
+            <Switch>
+              <Route path="/embed/:userid/:id" exact children={SketchViewEmbed} />
+              <Route path="/">
+                <SideNav collapsed={this.state.navCollapsed} toggleNav={this.toggleNav} />
+                <main
+                  onClick={(e) => {
+                    if (!this.state.navCollapsed) {
+                      this.closeNav()
+                      e.preventDefault()
+                      return
+                    }
+                  }}
+                >
+                  <Header toggleNav={this.openNav} />
+                  <Switch>
+                    {Routes.map((route, key) => {
+                      let children = route.component
+                      if (route.requiresLogin && !this.props.user) {
+                        children = <Redirect to="/login" />
+                      }
+                      return <Route key={key} path={route.path} exact={route.exact} children={children}></Route>
+                    })}
+                  </Switch>
+                  <Footer />
+                </main>
+              </Route>
+            </Switch>
```

Comment:
It looks like this change is just formatting. Please can you refrain from large block changes like this unless it's a major change? Harder to review when it looks like the whole file has been removed and re-added basically.

Reply:
Just noticed this was wrapped inside a switch so no worries on the large block change!